src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+import './TabsPage.scss';
+
+export interface TypeTabs {
+  tabs: {
+    id: string;
+    title: string;
+    content: string;
+  }[];
+}
+
+export const TabsPage = ({ tabs }: TypeTabs) => {
+  const math: {params: {tabId: string}} = useRouteMatch();
+
+  const comparisonTabId = (tabId: string) => {
+    return `:${tabId}` === math.params.tabId;
```

Comment:
```suggestion
    return `${tabId}` === math.params.tabId;
```
Убери из адреса `:` 
Адрес должен выглядеть вот так
![image](https://user-images.githubusercontent.com/38065883/126984104-dc151a61-5992-4ce3-a1dc-4b763527313b.png)